data/defaults.yaml

```diff
@@ -57,7 +57,7 @@ prometheus::alertmanager::user: 'alertmanager'
 prometheus::alertmanager::version: '0.5.1'
 prometheus::alerts: {}
 prometheus::config_dir: '/etc/prometheus'
-prometheus::config_mode: '0660'
+prometheus::config_mode: '0640'
```

Comment:
@anarcat If you change the permissions, you need to modify the file ```spec/classes/prometheus_spec.rb``` aswell.

Reply:
thanks for the pointer! i've updated the test suite, which should hopefully reduce the number of failures :)

Reply:
i split this question off into a separate, more elaborate PR in #324